networks/injective/.env.development.example

```diff
@@ -0,0 +1,3 @@
+
+# can not start with 0x, so remove 0x from the beginning
```

Comment:
should this include? Has it put in gitignore?

Reply:
This is a .example file, deleting it is a wise choice, thanks for your advise